browser/main/Main.js

```diff
@@ -297,6 +338,39 @@ class Main extends React.Component {
         onMouseMove={(e) => this.handleMouseMove(e)}
         onMouseUp={(e) => this.handleMouseUp(e)}
       >
+      <div className={config.is_win ? 'windows-app-header' : 'mac-app-header'} styleName={config.is_win ? 'windows-app-header' : 'mac-app-header'}>
```

Comment:
why did you writer both `className` and `styleName`?
I think `className` is unnecessary.

Reply:
The same applies to other components.